docs/user-documentation/users.md

```diff
@@ -0,0 +1,36 @@
+# Managing Users
+
+## How to Add a User
+1. Log in to Drupal under admin credentials.
```

Comment:
Suggested change:
`Log in to Drupal under admin credentials.`
To:
Log into Drupal.  You will need the Administrator role to Manage Users.